lib/rules/jsx-uses-vars.js

```diff
@@ -33,6 +36,10 @@ module.exports = {
         if (node.name.name) {
           // <Foo>
           name = node.name.name;
+          // Exclude lowercase tag names like <div>
+          if (isTagName(name)) {
```

Comment:
what does this do with namespaced tags? Is `<a.X />` not allowed?

Reply:
`a.X` is not handled on this codepath, I believe that's under the condition below for `node.name.object`. [They're interpreted as references](https://babeljs.io/repl#?browsers=ie%2011&build=&builtIns=false&corejs=3.6&spec=false&loose=false&code_lz=DwPgUABBwGYPZwgenFYAxBzXXnAdAEICGATtpLgvgEZkXApA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=react&prettier=true&targets=&version=7.15.5&externalPlugins=%40babel%2Fplugin-syntax-object-rest-spread%407.8.3&assumptions=%7B%7D) so they should work fine.

If you would like, I could add the following valid test cases.

```js
    {
      code: `
        /* eslint jsx-uses-vars: 1 */
        var object;
        React.render(<object.Tag />);
      `
    }, {
      code: `
        /* eslint jsx-uses-vars: 1 */
        var object;
        React.render(<object.tag />);
      `
    }
```

Reply:
Thanks, adding those would be great :-)